src/rules/no-truthy-falsy.ts

```diff
@@ -29,9 +29,9 @@ export default createRule({
         }
 
         context.report({
-          data: { methodName: matcher.name }, // todo: rename to 'matcherName'
-          messageId: 'avoidMessage', // todo: rename to 'avoidMatcher'
+          messageId: 'avoidMatcher',
           node: matcher.node.property,
+          data: { matcherName: matcher.name },
```

Comment:
keep it alphabetical?

Reply:
We typically have it in this order from what I saw - I personally prefer this order as `messageId` & `node` are required and so will be present for all calls to `context.report`, then followed by `data` which is optional.